Romina.Api/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Data.SqlClient;
+using System.Linq;
+using Dapper;
+using Romina.Api.Models;
+
+namespace Romina.Api.Repositories
+{
+    public class ProductRepository: IProductRepository
+    {
+        public Product GetProductById(string id)
+        {
+            Product product = null;
+            using (SqlConnection connection = new SqlConnection("localConnectionString"))
+            {
+                var products = connection.Query<Product>("SELECT * FROM PRODUCT WHERE ID= @id",id).ToList();
+                product = products[0];
+            }
+
+            return product;
```

Comment:
following the TDD approach we really want to write tests before adding implementation 

but this we can go through together